stories/SQFormTextField.stories.tsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 import * as Yup from 'yup';
+import {Story, Meta} from '@storybook/react';
 
 import {SQFormTextField as SQFormTextFieldComponent} from '../src';
-import getSizeProp from './utils/getSizeProp';
+import getSizeProp, {gridOptions} from './utils/getSizeProp';
 import {createDocsPage} from './utils/createDocsPage';
-import {SQFormStoryWrapper} from './components/SQFormStoryWrapper';
+import {
+  SQFormStoryWrapper,
+  SQFormStoryWrapperProps,
+} from './components/SQFormStoryWrapper';
+import {SQFormTextFieldProps} from '../src/components/SQForm/SQFormTextField';
+
+type SQFormTextFieldStoryType = Story<
```

Comment:
So this definitely seems weird plus will probably need to be made reuseable in some way.

Basically our stories needs to accept the props for the component that they're for minus the size prop as that should be coming from storybook args.
They also needs to accept all the SQFormSToryWrapper props so we can supply initialValues and validation schema etc.

Reply:
Yeah, the story wrapper is creating a bit of a mess with types. I've been wanting to revisit that and see if there's another way to do it without the story wrapper types muddying the component types, while also allowing us to easily change SQForm props for the tests